### ISSUE TITLE
Add section for relevant dogecoin core contribution stats

### ISSUE THREAD

**Issue by nickatnight** [dogecoin/dogecoin.com#141]
Just recently came across this on [Sia](https://sia.tech/) coin site and think it would be a good idea to incorporate in doge. Gives full transparency on the development efforts

<img width="1397" alt="Screen Shot 2021-04-01 at 11 02 35 AM" src="https://user-images.githubusercontent.com/8783431/113335750-3b478200-92da-11eb-9b7e-0e4aa26e1024.png">

**Comment by edswf1** [comment]
i'll take a look at the github api

**Comment by nickatnight** [comment]
@edswf1 there should be a JS wrapper for their api that supports this right out the box...[here](https://github.com/octokit/rest.js/)

Unless @patricklodder prefers an in house solution to decrease the amount of libs we are loading

**Comment by edswf1** [comment]
> @edswf1 there should be a JS wrapper for their api that supports this right out the box...[here](https://github.com/octokit/rest.js/)
> 
> Unless @patricklodder prefers an in house solution to decrease the amount of libs we are loading

i don't think a library is needed, this simple function should do it:
``` javascript
function getFromGHAPI(source){
  let baseUrl = 'https://api.github.com/repos/dogecoin/dogecoin/';
  $.get(baseUrl + source + '?per_page=1').done(function(ret, responseText, jqXHR){
    try{
      var linkText = jqXHR.getResponseHeader('link');
      var last = linkText.split(',')[1];
      var count = last.split('&page=')[1].replace(/[^0-9]+/, '');
      console.log(count);
    }
    catch(e){
      console.error(e);
    }
  });
}
getFromGHAPI('commits'); //get number of commits -> 13532
getFromGHAPI('forks'); //get number of forks -> 1157
getFromGHAPI('releases'); //get number of releases -> 36
getFromGHAPI('contributors'); //get number of forks -> 349
```

in Detail: 
when per_page parameter is set, GH API returns a 'link' response header with next and last page. As we request only one item per page, the number of the last page is also the number of total elements available by GH API.

This is apparently not what GH uses to display numbers in the repository as there are 201 Contributors and 159 Releases there.

Edit: Seems like only published releases are available through GH API. No idea about contributors count though. But hey, numbers are close to the truth ;)

Edit 2: as GH API Limits Calls to 60/h/IP, a fallback and caching of those numbers would be needed

**Comment by nickatnight** [comment]
@edswf1 wow this is real elegant. Great job!

>as GH API Limits Calls to 60/h/IP, a fallback and caching of those numbers would be needed

[Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) my guy

**Comment by edswf1** [comment]
> [Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) my guy

true. i was just trying to say that the function is by no means complete yet

**Comment by nickatnight** [comment]
Hmm actually I'm not sure that would work because traffic to the site definitely has potential for more than 60 users per hour. Maybe create some routine to write those metrics your functions pull into a readable assets file? Then read those values and display.

Then, I think we can use local storage to prevent reading from file on every page load.

We could actually take this a step further and create a github action to write those stats to file based on some trigger...but that's probably too brute force for right now. I'd like to hear input from others first.

**Comment by edswf1** [comment]
> Hmm actually I'm not sure that would work because traffic to the site definitely has potential for more than 60 users per hour.

As i understood, this limit is per client IP. I think putting current values as a fallback would cover most errors then

**Comment by nickatnight** [comment]
>As i understood, this limit is per client IP

Oh ok per client ip..yeah that should be fine

**Comment by magneticat** [comment]
Am I wrong in assuming that due to the way Dogecoin Core is being developed such a graphic would not reflect the real amount of work being done on Doge? Patrick or Sporklin mentioned on r/dogecoindev (cannot find the thread now, thanks to reddit's beautiful search) that the github repo shows little activity because they develop the major releases in a repository not linked directly to github.

Correct me if I am wrong!

**Comment by nickatnight** [comment]
@magneticat I'm not actually sure about that! But where ever the repo lays, I'm sure we can pull stats from it. @patricklodder savvy?
